manager/controllers/app/moduleinstance.go

```diff
@@ -201,9 +247,10 @@ func GetSupportedReadSources(module *app.M4DModule) []*app.InterfaceDetails {
 	return list
 }
 
-// check whether copy is required
+// check whether IMPLICIT copy is required
```

Comment:
Please remove this comment.  This function isn't only for implicit copy

Reply:
It is. Changed the function name.